Xamarin.Forms.Platform.WPF/Renderers/ButtonRenderer.cs

```diff
@@ -36,6 +36,9 @@ protected override void OnElementChanged(ElementChangedEventArgs<Button> e)
 				if (Element.BorderWidth != 0)
 					UpdateBorderWidth();
 
+				if (Element.IsSet(Button.PaddingProperty))
```

Comment:
If you try to clear the padding property, it won't work because of this check.

Reply:
This (and the other one) are both in scope only for the first setup of the control, right?

Reply:
Sure, I was just testing you. Good job. 🥇 

😆